CIP-????/README.md

```diff
@@ -0,0 +1,311 @@
+---
+CIP: XXXX
+Title: Compensatory Ledger Mechanism
```

Comment:
```suggestion
Title: Governance Participant Compensation
```

I feel something referring to Governance Participants is probably more accurate and obvious at first glance

Reply:
yes... my first reading of the title also (before I got around to reading the document) was that this was some accounting capability and using "compensatory" in the mathematical sense (rather than "compensation").